source/SymWorld.h

```diff
@@ -173,15 +207,33 @@ class SymWorld : public emp::World<Organism>{
   emp::DataFile & SetupSymIntValFile(const std::string & filename) {
     auto & file = SetupFile(filename);
     auto & node = GetSymIntValDataNode();
+    auto & node5 = GetFreeSymIntValDataNode();
```

Comment:
The free-living sym information should be in its own data file since if free-living is off, this would be a lot of extra tracking for something that isn't relevant.